src/HttpHeader.cc

```diff
@@ -341,7 +341,7 @@ HttpHeader::Isolate(const char **parse_start, size_t l, const char **blk_start,
 }
 
 int
-HttpHeader::parse(const char *buf, size_t buf_len, bool atEnd, size_t &hdr_sz)
+HttpHeader::parse(const char *buf, size_t buf_len, bool atEnd, size_t &hdr_sz, const Http::StatusCode code)
```

Comment:
Pass a reference to a ContentLengthInterpreter object instead. See the other parse() method comment for details.

Reply:
Done.